src/commands/generate-cr.ts

```diff
@@ -74,6 +74,16 @@ export default class GenerateCr extends Command {
       multiple: true,
       description: 'Accepted tag types, comma separated values. Accepted values are ' + [...GenerateCr.tagTypeStrategies.keys()].join(", ")
     }),
+    overwriteTenants: Flags.boolean({
+      char: 'v',
```

Comment:
we could remove the chars but that's just my opinion

Reply:
since it's not mandatory we can remove them if it isn't the first char of the word in order to avoid putting random chars. Try to see `stdout` flag in pack command.

Reply:
Done